src/main/java/com/redhat/rcm/offliner/Options.java

```diff
@@ -27,6 +27,8 @@
 public class Options
 {
 
+    public static final String DEFAULT_TYPE_MAPPING = "test-jar=jar:tests;ejb=jar;ejb-client=jar:client;maven-plugin=jar;java-source=jar:sources;javadoc=jar:javadoc";
+
```

Comment:
Also, it might be good to have a command line option to spit out the default mappings and quit. This way, the user could save the mappings and add to them, rather than having to enter a complete list himself without help.